File: backend-portifolio/Models/Video.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace backend_portifolio.Models
{
    public class Video
    {
        [Key]
        public long Id { get; set; }
        public Guid Guid { get; set; }
        public int Sequence {get; set;}
        public string UrlVideo {get; set; }
    }
}

File: portfolio-v3/src/domain/models/Video.ts
import { Section } from "./Section";

export class Video{
    public Id: number;
    public Guid: string;
    public Sequence: number;
    public UrlVideo: string;
    public Caption: string;
    public SectionId: number;
    public Section: Section;
    constructor() {
        this.Id = 0;
        this.Guid = "";
        this.Sequence = 0;
        this.UrlVideo = "";
        this.Caption = "";
        this.SectionId = 0;
        this.Section = new Section();
    }
}